src/components/Signal/Signal.tsx

```diff
@@ -85,21 +44,89 @@ const Signal: React.FC<SignalProps> = ({
       {...props}
     >
       <title>{title}</title>
-      <circle cx="8" cy="8" fill={color} r="8" />
-      <g>
-        {paths.map((path, key) => (
-          <path
-            // eslint-disable-next-line react/no-array-index-key
-            key={`${kind}-icon-${key}`}
-            d={path}
-            fill={colors.neutral[0]}
-          />
-        ))}
+      <g style={{ mixBlendMode: 'multiply' }}>
+        <path
+          d="M3.5 8C4.3125 8 5 8.6875 5 9.5V14.5C5 15.3438 4.3125 16 3.5 16C2.65625 16 2 15.3438 2 14.5V9.5C2 8.6875 2.65625 8 3.5 8Z"
```

Comment:
Please update bar paths according to Figma. They weren't centred in the viewbox and Mate fixed that.